buildbot/precommit_report.py

```
@@ -48,15 +48,13 @@ def trunctate_diff(st):
-        print(log_f)
-    elif f == "log_precommit_pragma_once_check":
-        print(log_f)
-    elif f == "log_precommit_doxygen_header":
-        print(log_f)
-    elif f == "log_precommit_check-Authorship-update":
+    if (
+        f == "log_precommit_check_sycl_include"
+        or f == "log_precommit_license_check"
+        or f == "log_precommit_pragma_once_check"
+        or f == "log_precommit_doxygen_header"
+        or f == "log_precommit_check-Authorship-update"
```

Ruff (0.16.0)

[warning] Consider merging multiple comparisons.

Merge multiple comparisons

(PLR1714)